tools/index-refresh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import sys
+
+from app import cli
+
+from AIPscan import db, typesense_helpers
+from config import CONFIGS
+
+
+def main():
+    # Initialize Flask app context
+    app = cli.create_app_instance(CONFIGS[cli.config_name], db)
+
+    with app.app_context():
+        if not typesense_helpers.typesense_enabled():
+            sys.exit("Error: Typesense not enabled in AIPscan.")
+
+        print("Initializing index...")
+        typesense_helpers.initialize_index()
+
+        print("Caching AIP data...")
+
+        completed_percent = None
+        for status in typesense_helpers.populate_index():
+            # Print update when percentage indexed (to nearest integer) has changed
+            if completed_percent != status["percent"]:
+                print(f"Indexing {status['type']} data ({status['percent']}%)")
+                completed_percent = status["percent"]
```

Comment:
I don't understand the `completed_percent` logic. Is it really needed? Why are you updating it? Could the if just check if `status["percent"]` is `None`.

Reply:
Same as you did in the index task.

Reply:
In this case it's useful as it only outputs progress when the percentage completed changes.